databuilder/models/table_elasticsearch_document.py

```diff
@@ -10,11 +10,11 @@ class TableESDocument(ElasticsearchDocument):
     def __init__(self,
                  database,  # type: str
                  cluster,  # type: str
-                 schema_name,  # type: str
+                 schema,  # type: str
                  name,  # type: str
                  key,  # type: str
                  description,  # type: str
-                 last_updated_epoch,  # type: Optional[int]
+                 last_updated_timestamp,  # type: Optional[int]
```

Comment:
If I understand it correctly, these two are only changes that actually change the payload and backward incompatible, right?

Reply:
That's correct! Thanks for testing it out!

You'll also need to delete/migrate the ES index in order to accept the new indexes on these renamed fields, I believe.